Bio/SeqFeature.py

```diff
@@ -445,10 +445,10 @@ def translate(
 
         if start_offset not in [0, 1, 2]:
             raise ValueError(
-                "The start_offset must be 0, 1, or 2. "
-                "The supplied value is {}. Check the value "
-                "of either the codon_start qualifier or "
-                "the start_offset argument".format(start_offset)
+                f"The start_offset must be 0, 1, or 2. "
```

Comment:
`f"` is redundant on this line. Also a minor style point, should it be aligned like this:
```
f"a"
"b"
```
or
```
f"a"
 "b"
```

Reply:
Good catch, thank you - there was a flake8 plugin which caught this kind of pointless f-string, but it had problems with false positives.

The black alignment is the first of the two.